packages/ani-cli/build.sh

```diff
@@ -5,10 +5,12 @@ TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION=2.2
 TERMUX_PKG_SRCURL=https://github.com/pystardust/ani-cli/archive/refs/tags/v${TERMUX_PKG_VERSION}.tar.gz
 TERMUX_PKG_SHA256=973d335a75bd7f920c244000ad6b057f702fb37752e7bea1b5bcf038785ec925
-TERMUX_PKG_DEPENDS="aria2, curl, grep, mpv, openssl-tool, sed, gawk, ffmpeg"
+TERMUX_PKG_DEPENDS="aria2, curl, grep, openssl-tool, sed, gawk, ffmpeg"
 TERMUX_PKG_PLATFORM_INDEPENDENT=true
 TERMUX_PKG_BUILD_IN_SRC=true
 
 termux_step_make_install() {
 	install -Dm700 -t $TERMUX_PREFIX/bin ani-cli
+	echo 'am start --user 0 -a android.intent.action.VIEW -d "$1" -n is.xyz.mpv/.MPVActivity' > $PREFIX/bin/mpv
+	echo 'Note that you need to have the mpv android app installed'
```

Comment:
`$PREFIX/bin/mpv` is provided by the mpv and mpv-x packages, so this will cause ani-cli to conflict with those packages. A better solution could be to put this wrapper in `$TERMUX_PREFIX/libexec/mpv-android`, and change so that ani-cli sets $player_fn to that script if termux is detected

The info message is only displayed during the build, so users won't see it, maybe you want to echo it in a postinst script, as done by for example [texlive-installer](https://github.com/termux/termux-packages/blob/master/packages/texlive-installer/build.sh#L53-L63)